app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_args = command.split(" ")
```

Comment:
`" "` is default value for split
```suggestion
    command_args = command.split()
```